.gitignore

```diff
@@ -80,7 +80,6 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
 
```

Comment:
You should add files not related to the project to `.gitignore`, example of this is the `.DSTORE` file.